### ISSUE TITLE
[Question] Experience with segmentation downstream tasks

### ISSUE THREAD

**Issue by RGring** [facebookresearch/swav#54]
Hi Mathilde,
I was wondering if you have performed experiments, using swav pretrained weights on segmentation downstream tasks. If yes, do you get similar performance as on classification and detection?

I would appreciate any insights on that matter!

**Comment by mathildecaron31** [comment]
Hi @RGring 

Sorry for the delay of my reply. I have experimented with instance segmentation finetuning on COCO with Mask-RCNN. See table 7 (in appendix) for detailed results on this benchmark. I have not taken the time to try other segmentation benchmark unfortunately.
